packages/kununu-theme-v3/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@kununu/kununu-theme-v3",
+  "version": "3.0.0-beta.0",
```

Comment:
Should we publish under beta for now? At least until we add all basic CSS we need for this project.